src/components/BetCreatorForm.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import BetEntry from '../models/BetEntry'
+import { BetDataContext } from '../providers/BetDataProvider'
+
+const newId = -1
+interface Props {
+  onBetAdded: (bet: BetEntry) => void
+}
+const BetCreatorForm: React.FC<Props> = ({ onBetAdded }) => {
+  const [request, setRequest] = React.useState<BetEntry>(
+    new BetEntry(newId, '', '', '', '')
+  )
+  const betConsumer = React.useContext(BetDataContext)
+
+  const addBet = async () => {
+    try {
+      const response = await betConsumer.addNewBet(request)
+      onBetAdded(response)
+    } catch (e: any) {
+      console.error('failed')
+    }
+  }
+  return (
+    <div className="form flex space-between">
+      <textarea
```

Comment:
We can use TextField from [FluentUI library](https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield)

Reply:
maybe next time